prow/istio-presubmit.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -ex
+
+E2E_ARGS=()
+
+if [ "${CI}" == "bootstrap" ]; then
+    # ensure correct path
+    mkdir -p ${GOPATH}/src/istio.io
```

Comment:
GOPATH is set in the docker image. https://github.com/istio/test-infra/blob/master/docker/prowbazel/Dockerfile#L43